skdecide/hub/domain/up/up.py

```diff
@@ -573,6 +683,36 @@ def _get_observation_space_(self) -> D.T_agent[Space[D.T_observation]]:
                         else np.int32
                     ),
                 )
+            elif self._state_encoding == "variable":
+                self._observation_space = VariableSpace(
+                    Box(
```

Comment:
@HamdaHmida Can you please use the scikit-decide version of gym.Box, i.e. BoxSpace?